akka-stream/src/main/boilerplate/akka/japi/tuple/Tuple.scala.template

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.tuple
```

Comment:
Actually there isn't any yet, so we might want to migrate this there eventually.